### PR TITLE
Added options to create index statement

### PR DESCRIPTION
**Pull request by madams51703**

### CODE REVIEW

File: plsql/PlSqlParser.g4 [hunk 1]
```diff
@@ -521,9 +521,145 @@ sequence_start_clause
...
+index_partitioning_clause
+    : PARTITION partition_name? VALUES LESS THAN '(' 
+                                        (','? (CHAR_STRING
+                                              | string_function
+                                              | numeric
+                                              | MAXVALUE
+                                              )
+                                        )+ 
+                                                 ')'
+                                        segment_attributes_clause?
```
Comment (KvanTTT) [inline plsql/PlSqlParser.g4]: Could such big indent be reduced?

**Comment by madams51703** [comment]
Yes, I already had created a literal rule in my next release and started to
use it.  I can go and modify these large indents to use the literal rule.
I have the next release ready to queue up pending reviewing formatting.
Can I fix this with the pending release since I have already started down
that road?

**Comment by KvanTTT** [comment]
Ok then!

**Comment by madams51703** [comment]
Thank you!

**Pull request merged**